src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -136,13 +138,31 @@ public String getProjectName() {
         return projectName;
     }
 
+    /**
+     * Sets directory qualifier.
+     *
+     * @param directorySuffix directory qualifier
+     */
+    public void setdirectorySuffix(String directorySuffix) {
+        this.directorySuffix = directorySuffix;
+    }
+
+    /**
+     * Returns directory qualifier
+     *
+     * @return directory qualifier
+     */
+    public String getdirectorySuffix() {
+        return StringUtils.defaultString(directorySuffix);
+    }
+
     /**
      * Gets directory where the attachments are stored.
      *
      * @return directory for attachment
      */
     public File getEmbeddingDirectory() {
-        return new File(getReportDirectory().getAbsolutePath(), ReportBuilder.BASE_DIRECTORY
+        return new File(getReportDirectory().getAbsolutePath(), ReportBuilder.BASE_DIRECTORY + this.getdirectorySuffix()
```

Comment:
IMO it would be nice to have some separator such as underscore between BASE_DIRECTORY and suffix. Consider this

Reply:
I'd rather let the end user decide whether such separator is needed - if yes, then it can be passed as part of the suffix (as I do myself in my own compiled version).